src/opcode.rs

```diff
@@ -1832,3 +1832,159 @@ opcode! {
         Entry(sqe)
     }
 }
+
+// === 6.8 ===
+
+opcode! {
+    /// Install a fixed file descriptor
+    ///
+    /// Turns a direct descriptor into a regular file descriptor that can be later used by regular
+    /// system calls that take a normal raw file descriptor
+    #[derive(Debug)]
+    pub struct FixedFdInstall {
+        fd: { types::Fixed },
+        file_flags: { u32 },
+        ;;
+    }
+
+    pub const CODE = sys::IORING_OP_FIXED_FD_INSTALL;
+
+    pub fn build(self) -> Entry {
+        let FixedFdInstall { fd, file_flags } = self;
+
+        let mut sqe = sqe_zeroed();
+        sqe.opcode = Self::CODE;
+        sqe.fd = fd.0 as _;
+        sqe.flags = 1 << sys::IOSQE_FIXED_FILE_BIT;
+        sqe.__bindgen_anon_3.install_fd_flags = file_flags;
+        Entry(sqe)
+    }
+}
+
+// === 6.9 ===
+
+opcode! {
+    /// Perform file truncation, equivalent to `ftruncate(2)`.
+    #[derive(Debug)]
+    pub struct FTruncate {
```

Comment:
```suggestion
    pub struct Ftruncate {
```

To be consistent with `Fallocate` and `Fadvise`.

Reply:
Updated PR to resolve this